backend/middleware/requestLimiter.ts

```diff
@@ -0,0 +1,33 @@
+import RateLimit from 'express-rate-limit'
+import RedisStore from 'rate-limit-redis'
+import redis from 'redis'
+
+import config from '../config'
+
+// If more than 10 requests are made to the same url and ip address within 30 seconds
+// the rate limiter will kick in and the user will receive "Too many requests, please try again later."
+// and an HTTP 429 status code
+export default (maxConnections?) =>
+  new RateLimit({
+    ...initRedisStoreIfEnabled({
```

Comment:
What happens if redis isn't enabled? Does it just keep a tally of requests for the server its on?